src/Microsoft.AspNet.Identity.EntityFramework/UserStore.cs

```diff
@@ -50,6 +70,11 @@ public class UserStore<TUser, TRole, TContext, TKey> :
         where TKey : IEquatable<TKey>
     {
 
+        /// <summary>
+        /// Creates a new instance of <see cref="UserStore"/>.
+        /// </summary>
+        /// <param name="context">The context used to access the store.</param>
+        /// <param name="describer">The <see cref="IdentityErrorDescriber"/> to place store errors into.</param>
```

Comment:
Not sure this is the best description. Perhaps "used to obtain descriptions for errors returned by the store"? @haok?